### PR TITLE
[IMP] industry_restaurant: Improving flows

### PR DESCRIPTION
**Pull request by jaeschwa**
Improving the flows of this industry.

- Uploaded restaurant config (no bar), linked appointments with floor plan, renamed to "Book a table".  
- Added knowledge article, "Blond Beer" with reordering, UOM, supplier update.  
- Removed contacts "Rio Pone" and "Roger Scott", fixed knowledge article separators.  
- Installed Purchase, Planning, Project, and Contacts apps.  
- Uninstalled Barcode, Sales, Ecommerce, and Inventory apps.  
- Removed SO lines, stock moves, pickings, and branches.  
- Created "Ecodys", "V.D.S.-FOOD", products, and POs (PO0001, PO0002).  
- Updated website, linked "Book a table", and loaded restaurant POS scenario.

task-4069767

### CODE REVIEW

File: industry_restaurant/__manifest__.py [hunk 1]
```diff
@@ -9,28 +9,27 @@
     'depends': [
         'account_followup',
         'knowledge',
+        'contacts',
         'pos_enterprise',
+        'pos_restaurant',
+        'project',
+        'purchase_stock',
+        'planning',
+        'hr_employee',
```
Comment (vava-odoo) [inline industry_restaurant/__manifest__.py:18]: alphabetical order please

File: industry_restaurant/data/appointment_type.xml [hunk 2]
```diff
@@ -1,52 +1,58 @@
...
+        (0, 0, {'weekday': '3', 'start_hour': 13.0, 'end_hour': 15.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 18.0, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 18.25, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 18.5, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 18.75, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 19.0, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 19.25, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 19.5, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '3', 'start_hour': 19.75, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '4', 'start_hour': 12.0, 'end_hour': 15.0})
```
Comment (vava-odoo) [inline industry_restaurant/data/appointment_type.xml:55]: not weekday 4 and 5?

File: industry_restaurant/data/appointment_type.xml [hunk 3]
```diff
@@ -1,52 +1,58 @@
...
+        (0, 0, {'weekday': '1', 'start_hour': 12.75, 'end_hour': 15.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 13.0, 'end_hour': 15.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 18.0, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 18.25, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 18.5, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 18.75, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 19.0, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 19.25, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 19.5, 'end_hour': 23.0}),
+        (0, 0, {'weekday': '1', 'start_hour': 19.75, 'end_hour': 23.0}),
```
Comment (vava-odoo) [inline industry_restaurant/data/appointment_type.xml:28]: Also  `(0, 0, {'weekday': '1', 'start_hour': 20.0, 'end_hour': 23.0}),`?

File: industry_restaurant/data/product_category.xml [hunk 4]
```diff
@@ -32,4 +32,12 @@
         <field name="name">Combo Offers</field>
         <field name="parent_id" ref="product.product_category_all"/>
     </record>
+    <record id="product_category_15" model="product.category">
+        <field name="name">Deliveries</field>
+        <field name="parent_id" ref="product.product_category_all"/>
+    </record>
```
Comment (vava-odoo) [inline industry_restaurant/data/product_category.xml:38]: Seems like it is not used

File: industry_restaurant/data/product_product.xml [hunk 5]
```diff
@@ -1,205 +1,49 @@
...
 <odoo noupdate="1">
-    <record id="product_product_5" model="product.product">
-        <field name="name">Gift Card</field>
-        <field name="type">service</field>
-        <field name="list_price">0.0</field>
+    <record id="product_product_54" model="product.product">
+        <field name="name">Beef 1kg</field>
+        <field name="type">consu</field>
         <field name="sale_ok" eval="False"/>
+        <field name="list_price">1</field>
```
Comment (vava-odoo) [inline industry_restaurant/data/product_product.xml:7]: consu and price of 1 are the default
```suggestion
        <field name="sale_ok" eval="False"/>
```

File: industry_restaurant/data/project_project.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+<odoo noupdate="1">
+    <record id="project_project_1" model="project.project">
+        <field name="name">Suppliers Orders</field>
+        <field name="stage_id" ref="project.project_project_stage_0"/>
+        <field name="type_ids" eval="[(6, 0, [ref('project_task_type_1'), ref('project_task_type_2')])]"/>
+    </record>
+    <record id="project_project_2" model="project.project">
+        <field name="name">Cleaning</field>
+        <field name="stage_id" ref="project.project_project_stage_0"/>
+        <field name="type_ids" eval="[(6, 0, [ref('project_task_type_3'), ref('project_task_type_4')])]"/>
```
Comment (vava-odoo) [inline industry_restaurant/data/project_project.xml:11]: Can't you share the `project.task.type`? (and get rid of 3 and 4)
```suggestion
        <field name="type_ids" eval="[(6, 0, [ref('project_task_type_1'), ref('project_task_type_2')])]"/>
```

File: industry_restaurant/data/res_config_settings.xml [hunk 7]
```diff
@@ -3,6 +3,11 @@
     <record id="res_config_settings_enable_restaurant" model="res.config.settings">
         <field name="qr_code" eval="1"/>
         <field name="group_product_variant" eval="1"/>
+        <field name="group_project_recurring_tasks" eval="1"/>
+        <field name="pos_use_pricelist" eval="1"/>
+        <field name="group_stock_packaging" eval="1"/>
+        <field name="pos_self_ordering_mode" eval="'mobile'"/>
+        <field name="group_uom" eval="1"/>
```
Comment (vava-odoo) [inline industry_restaurant/data/res_config_settings.xml:10]: You don't define new one, why do you need it?

File: industry_restaurant/demo/kitchen_display.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+<record id="preparation_display_main_restaurant_bar" model="pos_preparation_display.display">
+    <field name="name">Bar Display</field>
+    <field name="pos_config_ids"
+    eval="[(6, 0, [ref('pos_restaurant.pos_config_main_restaurant')])]" />
```
Comment (vava-odoo) [inline industry_restaurant/demo/kitchen_display.xml:6]:
```suggestion
    <field name="pos_config_ids" eval="[(6, 0, [ref('pos_restaurant.pos_config_main_restaurant')])]" />
```

File: industry_restaurant/demo/kitchen_display.xml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+<record id="preparation_display_main_restaurant_bar" model="pos_preparation_display.display">
```
Comment (vava-odoo) [inline industry_restaurant/demo/kitchen_display.xml:3]:
```suggestion
    <record id="preparation_display_main_restaurant_bar" model="pos_preparation_display.display">
```

File: industry_restaurant/demo/product_product.xml [hunk 10]
```diff
@@ -1,54 +1,32 @@
...
+        <record id="product_product_56" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_3')])]"/>
+        </record>
+        <record id="product_product_57" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_4')])]"/>
+        </record>
+        <record id="product_product_58" model="product.product">
+            <field name="seller_ids" eval="[(6, 0, [ref('product_supplierinfo_5')])]"/>
+        </record>
+        <record id="product_product_59" model="product.product">
```
Comment (vava-odoo) [inline industry_restaurant/demo/product_product.xml:18]: Most of its definition should be in data

File: industry_restaurant/static/src/binary/ir_attachment/knowledge_cover.jpg
Comment (vava-odoo) [inline industry_restaurant/static/src/binary/ir_attachment/knowledge_cover.jpg:1]: new images are quite heavy. Could you reduce?

**Review by vava-odoo — COMMENTED** [review]
Thanks @jaeschwa 
I've listed a few comments, mostly nitpicking.
Although I find this demo industry great, I feel like there is nothing in data for production databases. My view is that a significant part of the files should be moved to data. Otherwise, we lose what was configured before.

File: industry_restaurant/demo/product_product.xml (hunk 10 above)
Comment (jaeschwa) [reply industry_restaurant/demo/product_product.xml:18]: It cannot the supplier require partner.

File: industry_restaurant/data/res_config_settings.xml (hunk 7 above)
Comment (jaeschwa) [reply industry_restaurant/data/res_config_settings.xml:10]: What are you referring to ?

File: industry_restaurant/demo/kitchen_display.xml [hunk 11]
```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="preparation_display_main_restaurant_bar" model="pos_preparation_display.display">
+        <field name="name">Bar Display</field>
+        <field name="pos_config_ids" eval="[(6, 0, [ref('pos_restaurant.pos_config_main_restaurant')])]" />
+        <field name="category_ids" model="pos.category" eval="[(6, 0, obj().search([('name', 'ilike', 'Drinks')]).ids)]"/>
```
Comment (vava-odoo) [inline industry_restaurant/demo/kitchen_display.xml:6]:
```suggestion
        <field name="category_ids" model="pos.category" search="[('name', 'ilike', 'Drinks')]"/>
```

File: industry_restaurant/demo/product_product.xml (hunk 10 above)
Comment (vava-odoo) [reply industry_restaurant/demo/product_product.xml:18]: Yes, some parts should remain here, hence the "most of" in my comment ;-)

File: industry_restaurant/__manifest__.py (hunk 1 above)
Comment (vava-odoo) [reply industry_restaurant/__manifest__.py]: almost :nerd_face:

**Review by vava-odoo — APPROVED** [review]
After discussion, it's ok :+1: 
Just a few nitpicking comments and we have a new module!
Could you also fix the commits?
Thanks! :muscle:

**Pull request merged**